docs/ci/github.mdx

```diff
@@ -173,3 +173,184 @@ The `shorebird-patch` action also outputs the patch number:
 ```
 
 :::
+
+# 🫸 Push to Deploy
```

Comment:
```suggestion
# 🤖 Fully Automated Workflow Example
```

I wonder if we shouldn't just refer to this as a complete automated workflow since I'm not sure whether or not people will know what "Push to Deploy" means especially now that this is part of the GitHub docs.

Reply:
Good call, I will rename other references in the text to match this suggestion.